tools/qtc_templates/kuesa/project/empty_quick/app.pro

```diff
@@ -0,0 +1,19 @@
+QT += quick
+CONFIG += resources_big
+
+DEFINES += QT_DEPRECATED_WARNINGS
+
+SOURCES += \\
+        %{CppFileName}
```

Comment:
Indent with only four spaces here? That's what you used for the main.qml below.

Reply:
done